charts/vector/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: vector
-version: "0.1.0-alpha.1"
-kubeVersion: ">=1.15.0"
+version: "0.1.0-alpha.2"
+kubeVersion: ">=1.15.0-0"
```

Comment:
This would fail when attempting to install against an EKS cluster:

```shell
Error: UPGRADE FAILED: chart requires kubeVersion: >=1.15.0 which is incompatible with Kubernetes v1.20.7-eks-d88609
```